group_vars/matrix_servers

```diff
@@ -18,8 +18,6 @@
 
 matrix_identity_server_url: "{{ ('https://' + matrix_server_fqn_matrix) if matrix_ma1sd_enabled else None }}"
 
-matrix_riot_jitsi_preferredDomain: "{{ matrix_server_fqn_jitsi if matrix_jitsi_enabled else '' }}"
```

Comment:
Why not rename this to `matrix_client_element_jitsi_preferredDomain` instead?

Reply:
Because `matrix_client_element_jitsi_preferredDomain` already existed in `group_vars/matrix_servers`. It was in twice, once used by `roles/matrix-base/templates/static-files/well-known/matrix-client.j2`, the other by `roles/matrix-client-element/templates/config.json.j2`. I assumed this was a bug.

Reply:
You're right! Seems like it is a bug!